handlers/user/search.go

```diff
@@ -0,0 +1,22 @@
+package user
+
+import (
+	"net/http"
+
+	contacts "github.com/wshaman/contacts-stub"
+	"github.com/wshaman/course-rest/lib"
+)
+
+func Search(w http.ResponseWriter, r *http.Request) {
+	// url transfer as a parameter
+	query := r.URL.Query()
+	number := query.Get("number")
+	p := contacts.LoadPersistent()
```

Comment:
What should be in case when we have no contacts returned by 'LoadPersistent' ?

Reply:
we already check len in FindByPhone.
Maybe we can call method List before FindByPhone